cfe_internal/enterprise/CFE_knowledge.cf

```diff
@@ -317,14 +317,6 @@ bundle agent cfe_internal_setup_knowledge
         perms => mog("0440", "root", $(def.cf_apache_group) ),
         comment => ".htaccess files should only be readable by webserver.";
 
-      "$(cfe_internal_hub_vars.docroot)/ldap/config/settings.ldap.php" -> { "ENT-3400" }
```

Comment:
nope. needs write permissions and we enforce rx on everything with the ldap directory recursive promise, so this is still needed, just need to make an exception in the directory promise.